src/js/common/components/NavHeader.jsx

```diff
@@ -1,30 +1,22 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import { getCurrentFormStep, getCurrentPageName } from '../utils/helpers';
+
 export default class NavHeader extends React.Component {
   render() {
     const { chapters, path, className } = this.props;
     const total = chapters.length;
 
-    let step;
-    let name;
-    chapters
-      .forEach((chapter, index) => {
-        if (chapter.pages.some(page => page.path === path)) {
-          step = index + 1;
-          name = chapter.name;
-        }
-      });
-
-    return step
+    return getCurrentFormStep(chapters, path)
       ? <h4
           role="progressbar"
-          aria-valuenow={step}
+          aria-valuenow={getCurrentFormStep(chapters, path)}
```

Comment:
These get calls seem repetitive and it might be better to set them to variables at the top of the render function.